sdk/core/azure-core/inc/azure/core/version.hpp

```diff
@@ -53,3 +53,4 @@ namespace Azure { namespace Core {
 #undef AZURE_CORE_VERSION_PATCH
 #undef AZURE_CORE_VERSION_PRERELEASE
 
+
```

Comment:
@weshaggard - Our EngSys script that updates versions needs to be fixed to not add this new line.

Reply:
@ahsonkhan how important is it to not have a newline at the end of this file? I will say it is pretty common for most files to end in a newline. If it is important then you will need to make some adjustments to https://github.com/Azure/azure-sdk-for-cpp/blob/master/eng/scripts/Update-PkgVersion.ps1#L60 so that it doesn't write a newline after the last line in the file.

Reply:
We end in a newline.  The issue is the script is adding 2.

Reply:
I'd have to double check but I believe you are ending in an empty line but that doesn't have a newline char on that line. I believe the script is only adding the newline char.